subprojects/extTest/extTest.gradle

```diff
@@ -1,3 +1,5 @@
+apply from: "$rootDir/gradle/dependencies.gradle"
```

Comment:
Is this line needed?

Reply:
Rather yes. There is:
```
dependencies {
    testCompile "org.mockito:mockito-core:$version"
    testCompile project(path: ':', configuration: 'testUtil')
    testCompile libraries.junit4
    testCompile libraries.assertj
}
```
in that file below.